src/util/system.cpp

```diff
@@ -399,6 +386,13 @@ std::optional<unsigned int> ArgsManager::GetArgFlags(const std::string& name) co
     return std::nullopt;
 }
 
+fs::path ArgsManager::GetPathArg(std::string pathlike_arg) const
```

Comment:
Other `Get...Arg` methods use the camelCase notation for arguments. Let's change to `pathlikeArg` or similar.

Reply:
https://github.com/bitcoin/bitcoin/pull/24265#discussion_r800153360